src/main/resources/rhizome.yaml

```diff
@@ -19,6 +19,21 @@ hazelcast:
         capacity: 128
         durability: 1
 
+hazelcast-clients:
+    IDS:
+        server: false
+        group: "local"
+        password: "trellis"
+        cp-member-count: 0
+        seed-nodes:
+            - "localhost"
+        scheduled-executor:
```

Comment:
The `scheduled-executor` block is safe to remove (including it will have no effect)

Reply:
Cool, thanks!